targets/TARGET_NUVOTON/TARGET_M480/objects.h

```diff
@@ -1,5 +1,5 @@
 /* mbed Microcontroller Library
- * Copyright (c) 2015-2016 Nuvoton
+ * Copyright (c) 2019 Nuvoton
```

Comment:
Would be preferred to follow the old format here - **2015-2019**

Reply:
Changed to **2015-2019**.